core/src/main/java/discord4j/core/event/domain/interaction/DeferrableInteractionEvent.java

```diff
@@ -201,8 +201,10 @@ public InteractionApplicationCommandCallbackReplyMono reply() {
      * Requests to respond to the interaction with a notification instructing the user that this interaction requires a premium subscription.
      *
      * @return A {@link Mono} where, upon successful completion, emits nothing; indicating the interaction response has been sent. If an error is received, it is emitted through the {@code Mono}.
+     * @deprecated in favor of using {@link discord4j.core.object.component.Button#premium(Snowflake)}. This will continue to function but may be eventually unsupported
      */
     @Experimental // This method could not be tested due to the lack of a Discord verified application
+    @Deprecated
```

Comment:
If is now deprecated i think not need the experimental annotation.

Reply:
Even if it's deprecated, if someone wants to use it, they should know that it's experimental